validation/validate/range.go

```diff
@@ -1,10 +1,33 @@
 package validate
 
 import (
-	"github.com/target/goalert/validation"
 	"strconv"
+
+	"github.com/target/goalert/validation"
 )
 
+// Len will ensure a slice has at least min and at most max items.
+func Len[T any](fname string, val []T, min, max int) error {
```

Comment:
Some other options from GPT:

- validateMapLen
- mapLenValidator
- checkMapLen
- ensureValidMapLen
- mapLenBounds
- mapLenCheck

Reply:
The way it will be used is `validate.Len` and the one below it is `validate.MapLen`

It's meant to play off of the existing built-in and provide a validation function that works in a similar way.

https://google.github.io/styleguide/go/best-practices#avoid-repetition